### PR TITLE
📝 Update request body tutorial note to reference 3.10+ syntax

### PR DESCRIPTION
**Pull request by graysonpike**
In the Request Body and Query Params sections of the tutorial, there is a Note which references the old 3.8 syntax instead of the 3.10+ syntax (which is shown by default). To reduce confusion, I have updated the note's syntax to match what is displayed by default.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit d8cf0801f724253965dc26ede3bf3ac9a0fc54ef at: https://7f7daa88.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 199ddfc2f6b752fa206c750ecb9f1bc798172a5f at: https://f8f60ad8.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit ec9f329654dfb7d9315f12047018a183ddbf23e4 at: https://dd78ef0b.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8a0fefc6283f77b29e94ea73d2021f119213c0c9 at: https://05e6baa9.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9c3ec48ad11e23e4e0b3bfc2285f324ed9b39dc6 at: https://cb3ad7a5.fastapitiangolo.pages.dev

**Review by svlandeg — COMMENTED** [review]
Hi, thanks for the PR! This was indeed phrased a bit confusingly before.

Since you created this PR, some edits to the docs have already been made to address this, including https://github.com/fastapi/fastapi/pull/11415. As such, I don't think we need to change much here anymore. It does look to me like there's been some duplication of text in these docs pages, so I've gone ahead and cleaned things up a little.

File: docs/en/docs/tutorial/body.md [hunk 1]
```diff
@@ -235,8 +235,6 @@ The function parameters will be recognized as follows:
 
 /// note
 
-FastAPI will know that the value of `q` is not required because of the default value `= None`.
```
Comment (svlandeg) [inline docs/en/docs/tutorial/body.md:238]: The same is said in the sentence right below this one.

File: docs/en/docs/tutorial/query-params-str-validations.md [hunk 2]
```diff
@@ -24,9 +24,7 @@ The query parameter `q` is of type `Union[str, None]` (or `str | None` in Python
 
 /// note
 
-FastAPI will know that the value of `q` is not required because of the default value `= None`.
```
Comment (svlandeg) [inline docs/en/docs/tutorial/query-params-str-validations.md:27]: The same is said in the sentence right above this one.

**Comment by graysonpike** [comment]
Hi @svlandeg, thank you for noticing the change! I wil close this PR as it is no longer needed.

**Pull request closed without merging**